Library/Homebrew/dev-cmd/create.rb

```diff
@@ -94,7 +94,7 @@ def create_cask(args:)
     token = Cask::Utils.token_from(name)
 
     cask_tap = Tap.fetch(args.tap || "homebrew/cask")
-    raise TapUnavailableError, args.tap unless cask_tap.installed?
+    raise TapUnavailableError, cask_tap unless cask_tap.installed?
```

Comment:
Not that it matters in practice, but this is more consistent with how this error is used elsewhere:

```suggestion
    raise TapUnavailableError, cask_tap.name unless cask_tap.installed?
```